nixos/modules/services/networking/bind.nix

```diff
@@ -211,25 +212,33 @@ in
 
     networking.resolvconf.useLocalResolver = mkDefault true;
 
-    users.users.${bindUser} =
-      {
+    users = {
+      groups = {
+        "${bindGroup}" = {
+          gid = config.ids.gids.bind;
```

Comment:
FWIW i have no idea how to declare this so it might fail at some point

Reply:
https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/misc/ids.nix#L93

Reply:
Resolved in https://github.com/NixOS/nixpkgs/pull/129656/commits/01941f460434a2110f59eaf341092ffe65bbc946